with your change, I still get the cpu fault

```
PASS - dns_query_ipv6_numeric
===================================================================
Test suite dns_tests succeeded
===================================================================
PROJECT EXECUTION SUCCESSFUL
***** CPU Page Fault (error code 0x00000002)
Supervisor thread wrote address 0x00000000
PDE: 0x025 Present, Read-only, User, Execute Enabled
PTE: 0x00 Non-present, Read-only, Supervisor, Execute Enabled
Current thread ID = 0x00401740
eax: 0x004016f0, ebx: 0x004016f0, ecx: 0x00000000, edx: 0x00000000
esi: 0x00000246, edi: 0x00000000, ebp: 0x0041c7d4, esp: 0x0041c7d4
eflags: 0x00000046 cs: 0x0008
call trace:
eip: 0x0000e999
     0x0000ec80 (0x41c7e0)
     0x00004a8e (0x2e)
     0x0000be5f (0x4160e4)
     0x00002c5e (0x0)
Fatal fault in ISR! Spinning...
Terminate emulator due to fatal kernel error
make[4]: *** [zephyr/CMakeFiles/run.dir/build.make:60: zephyr/CMakeFiles/run] Error 1
make[3]: *** [CMakeFiles/Makefile2:344: zephyr/CMakeFiles/run.dir/all] Error 2
make[2]: *** [CMakeFiles/Makefile2:351: zephyr/CMakeFiles/run.dir/rule] Error 2
make[1]: *** [Makefile:196: run] Error 2
make: *** [/home/nashif/bin/zb:34: run] Error 2

```

--- a/tests/net/lib/dns_resolve/src/main.c
+++ b/tests/net/lib/dns_resolve/src/main.c
@@ -617,6 +617,7 @@ static void dns_query_ipv4(void)
 	}
 }
 
+#if defined(TEMPORARILY_DISABLED_TEST)
 static void dns_query_ipv6(void)
 {
 	struct expected_status status = {
@@ -644,6 +645,7 @@ static void dns_query_ipv6(void)
 		zassert_true(false, "Timeout while waiting data");
 	}
 }
+#endif
 
 struct expected_addr_status {
 	struct sockaddr addr;
@@ -714,6 +716,7 @@ static void dns_query_ipv4_numeric(void)
 	}
 }
 
+#if defined(TEMPORARILY_DISABLED_TEST)
 static void dns_query_ipv6_numeric(void)
 {
 	struct expected_addr_status status = {
@@ -741,6 +744,7 @@ static void dns_query_ipv6_numeric(void)
 		zassert_true(false, "Timeout while waiting data");
 	}
 }
+#endif
 
 void test_main(void)
 {
@@ -759,9 +763,7 @@ void test_main(void)
 			 ztest_unit_test(dns_query_ipv4_cancel),
 			 ztest_unit_test(dns_query_ipv6_cancel),
 			 ztest_unit_test(dns_query_ipv4),
-			 ztest_unit_test(dns_query_ipv6),
-			 ztest_unit_test(dns_query_ipv4_numeric),
-			 ztest_unit_test(dns_query_ipv6_numeric));
+			 ztest_unit_test(dns_query_ipv4_numeric));
 
 	ztest_run_test_suite(dns_tests);
 }

